eclipse-che/che-server
# Add latest devfile versions support

### What does this PR do?
* Add missing devfile versions to the list of supported devfile versions
* Add corresponding jsons.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

fixes https://github.com/eclipse/che/issues/22753
### How to test this PR?

1. Deploy che from the PR image: `quay.io/eclipse/che-server:pr-637`
2. Start a workspace from a repository with devfile version `2.2.2`

See: workspace starts successfully.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
